### PR TITLE
Pd 493 create wg easy tutorial

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR creates the wg-easy application tutorial, updates the deprecated OpenVPN service articles with a pointer to the new tutorial.

This merges into 22.12.3_docs.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+aliases:
+tags:
+- scaleopenvpn
+---
+
+
+{{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
+
+This article provides instructions on migrating from the SCALE OpenVPN client and server services to the new WG-Easy application. 
+This application is an open source utility that provides fast incremental file transfers TrueNAS SCALE uses as a replacement for the deprecated SCALE OpenVPN services.
```
Comment (stavros-k) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:14]: I think here forgot to update after copy pasting? :)

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+weight: 68
+aliases:
+tags:
+- scaleopenvpn
+---
+
+
+{{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
+
+This article provides instructions on migrating from the SCALE OpenVPN client and server services to the new WG-Easy application.
```
Comment (stavros-k) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:13]: wg-easy is only server, not a client

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Variables you can add are listed in the table below.
+{{< expand "Environmental Variables" "v" >}}
+{{< truetable >}}
+| Variable | Description |
+|----------|-------------|
+| WD_DEVICE | Enter the interface name or ID for the ethernet device WireGuard traffic should forward through. |
+| WG_PRE_UP | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
+| WG_POST_UP | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
+| WG_PRE_DOWN | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
+| WG_POST_DOWN | See [config.js](https://github.com/WeeJeWel/wg-easy/blob/master/src/config.js#L19) for the default value. Default value is `...`. |
```
Comment (stavros-k) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:66]: Default value is not `...` (The default value is in the file in the link.) probably just remove the last sentence

**Review by stavros-k — COMMENTED** [review]
Left some comments here and there. small things mostly!
Nice work!

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:14]: yep... gone in the next commit.

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md (hunk 3 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:66]: gone in the next commit

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+{{< trueimage src="/images/SCALE/22.12/WgeasyApplicationName.png" alt="Install wg-easy Application Name and Version" id="2: Install wg-easy Application Name and Version" >}}
+
+Accept the default value or enter a name in **Application Name**.
+
+Enter the configuration settings. 
+Enter the public host name or IP of your VPN server in **Hostname or IP**.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyConfigSettings.png" alt="wg-easy Configuration Settings" id="3: wg-easy Configuration Settings" >}} 
+
+If you use or want to protect access to the WG-Easy web UI enter a password in **Password for WebUI**.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:44]:
```suggestion
If you use or want to protect access to the WG-Easy web UI, enter a password in **Password for WebUI**.
```

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+Accept the default values in **Persistent Keep Alive** and **Clients MTU**, or enter the values you want to use.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:46]:
```suggestion
Accept the default values in **Persistent Keep Alive** and **Clients MTU** or enter the values you want to use. 
```

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+Enter the public host name or IP of your VPN server in **Hostname or IP**.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyConfigSettings.png" alt="wg-easy Configuration Settings" id="3: wg-easy Configuration Settings" >}} 
+
+If you use or want to protect access to the WG-Easy web UI enter a password in **Password for WebUI**.
+
+Accept the default values in **Persistent Keep Alive** and **Clients MTU**, or enter the values you want to use. 
+To change the time the connection remains alive, enter a value in seconds in **Persistent Keep Alive**. When set to zero, connections are not kept alive. 
+
+Accept the default IPs in **Clients IP Address Range** and **Clients DNS Server**, or enter the IP addresses the client to uses.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:49]:
```suggestion
Accept the default IPs in **Clients IP Address Range** and **Clients DNS Server** or enter the IP addresses the client uses.
```

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+To specify allowed IP addresses, click **Add** to the right of **Allowed IPs** for each IP address you want to enter. If you do not specify allowed IPs, the application uses 0.0.0.0/0.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyConfigAddAllowIPAndEnvironments.png" alt="wg-easy Allowed IPs and Environments" id="4: wg-easy Allowed IPs and Environments" >}} 
+
+To specify environment variables, click **Add** to the right of **WG-Easy Environment** for each environment variable you want to add. 
+Variables you can add are listed in the table below.
+{{< expand "Environmental Variables" "v" >}}
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:57]: consistency with the paragraph above

```suggestion
{{< expand "Environment Variables" "v" >}}
```

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+To add additional host path volumes, click **Add** to the right of **Extra Host Path Volumes**. 
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyStorageAddExtraHostPathVolumes.png" alt="wg-easy Add Extra Host Path Volumes" id="6: wg-easy Add Extra Host Path Volumes" >}} 
+
+Enter the path in **Mount Path in Pod** where you want to mount the volume inside the pod. 
+Enter or browse to the host path for the WG-Easy application dataset.
+
+Accept the default port numbers in **WireGuard UDP Node Port for WG-Easy** and **WebUI Node Port for WG-Easy**. WireGuard always listens on 51820 inside the Docker container. Refer to the TrueNAS [default port list]({{< relref "DefaultPorts.md" >}}) for the list of assigned port numbers.
+To change the port numbers, enter a number within the range 9000-65535. 
+Check the list of TrueNAS [Default Ports]({{< relref "defaultports.md" >}}) to verify the port number number is not already assigned or in use.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:84]:
```suggestion
Check the list of TrueNAS [Default Ports]({{< relref "defaultports.md" >}}) to verify the port number is not already assigned or in use.
```

File: content/SCALE/SCALETutorials/SystemSettings/Services/OpenVPNSCALE.md [hunk 9]
```diff
@@ -4,14 +4,16 @@ description: "Provides configuration information for OpenVPN Client and Server s
...
- - scaleopenvpn
+- scaleopenvpn
 ---
 
 
 {{< toc >}}
 
 {{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
 
+To configure VPN service, go to **Apps > Available Applications** and locate the **[wg-easy]({{< relref "InstallWgeasyApp.md" >}}** application.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/SystemSettings/Services/OpenVPNSCALE.md:15]: missing close parentheses 
```suggestion
To configure VPN service, go to **Apps > Available Applications** and locate the [**wg-easy**]({{< relref "InstallWgeasyApp.md" >}}) application.
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]
forgot to hit request changes on review, see previous comments

File: content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+Enter or browse to select the mount path for the host path.
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyStorageEnableCustomHostPath.png" alt="wg-easy Add Custom Host Path" id="5: wg-easy Add Custom Host Path" >}} 
+
+To add additional host path volumes, click **Add** to the right of **Extra Host Path Volumes**. 
+
+{{< trueimage src="/images/SCALE/22.12/WgeasyStorageAddExtraHostPathVolumes.png" alt="wg-easy Add Extra Host Path Volumes" id="6: wg-easy Add Extra Host Path Volumes" >}} 
+
+Enter the path in **Mount Path in Pod** where you want to mount the volume inside the pod. 
+Enter or browse to the host path for the WG-Easy application dataset.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md:80]: best practice is to not substitute the UI element name and style for what they enter (the value), in this case a host path value goes into the **Host Path** field. I choose to not use the UI element here and wanted to say what the value is.

File: content/SCALE/SCALETutorials/SystemSettings/Services/OpenVPNSCALE.md (hunk 9 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/OpenVPNSCALE.md:15]: Not sure if this renders correctly so I'm going to apply bold outside of the [ ].

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**